ceph-pull-requests/config/definitions/ceph-pull-requests.yml

```diff
@@ -62,3 +62,22 @@
                   - ABORTED
               build-steps:
                 - shell: "sudo reboot"
+      - cobertura:
+          report-file: "src/pybind/mgr/dashboard/frontend/coverage/cobertura-coverage.xml"
+          only-stable: "true"
+          fail-no-reports: "true"
+          fail-unhealthy: "true"
+          fail-unstable: "true"
+          health-auto-update: "true"
+          stability-auto-update: "true"
+          zoom-coverage-chart: "true"
+          source-encoding: "Big5"
```

Comment:
I don't think there is a need to change from ASCII here, why is `Big5` required?

Reply:
I checked on few forums, some people mentioned about some garbage characters which were not properly converted to ASCII being showing in report. So, I went with Big5. But, I didnt test will ASCII myself. Thoughts?

Reply:
Sounds reasonable. We can always update this if it is needed later. Thanks for the explanation!